caom2utils/caom2utils/fits2caom2.py

```diff
@@ -4106,6 +4198,11 @@ def _get_common_arg_parser():
                         help='output of augmented observation in XML',
                         required=False)
 
+    parser.add_argument('--caom_namespace',
+                        help=('if this parameter is specified, over-ride the '
```

Comment:
Maybe add examples of `caom_namespace` in help otherwise we'll forget and need to dig into the code. Or maybe this should specify the caom2 version instead. What is easier to remember?

Reply:
Either is good with me - I picked the namespace at the time because it was what was required programmatically.

Using the version will require an if statement to convert to the namespace, but if you would prefer the version, because that's how the caom2 package is implemented, let me know, and I'll change the implementation.